sdk/eventhub/eventhubs-checkpointstore-blob/samples/v1/javascript/README.md

```diff
@@ -0,0 +1,69 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-event-hubs
+urlFragment: eventhubs-checkpointstore-blob-javascript
+---
+
+# Azure Event Hubs client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure Event Hubs in some common scenarios.
+
+| **File Name**                                                                 | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [createCustomPipeline.js][createcustompipeline]                               | Demonstrates how to create a custom `Pipeline` that the Storage Blob Container Client uses to change the API version used when communicating with the service. This may be useful in environments like Azure Stack which supports an older version of Storage service than is officially supported by the Storage Blob SDK.                                                                                                                                                                                                                |
```

Comment:
It would make more sense to me for one of the other samples to be first in the table. You can use the `@azsdk-weight` property in the sample docstring (next to `@summary`) to reorder these. Just give this sample a negative weight, or give one or both of the other a positive weight (they appear in decreasing order and the default weight is zero).

Reply:
This is a file with a helper function used in the other 2 samples. Should we even list it in the table? cc @chradek

Reply:
I say we ditch it from the readme!

Reply:
Done in https://github.com/Azure/azure-sdk-for-js/pull/15137/commits/4b962cc7c8488427f44e8e776daac708761289b8